foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -319,83 +321,82 @@ function alertFunction() {
 ```
 
 ```javascript
-// the JavaScript file 
+// the JavaScript file
 // METHODS 2 & 3
 function alertFunction() {
-  alert("YAY! YOU DID IT!");
+  alert("YAY! YOU DID IT!");
 }
-const btn = document.querySelector('#btn')
+const btn = document.querySelector("#btn");
 
 // METHOD 2
 btn.onclick = alertFunction;
 
 // METHOD 3
-btn.addEventListener('click', alertFunction);
+btn.addEventListener("click", alertFunction);
 ```
 
 Using named functions can clean up your code considerably, and is a *really* good idea if the function is something that you are going to want to do in multiple places.
 
-With all three methods we can access more information about the event by passing a parameter to the function that we are calling.  Try this out on your own machine:
+With all three methods, we can access more information about the event by passing a parameter to the function that we are calling. Try this out on your own machine:
 
 ```javascript
-btn.addEventListener('click', function (e) {
-  console.log(e);
+btn.addEventListener("click", function (e) {
+  console.log(e);
 });
 ```
 
 <div class="lesson-note lesson-note--tip" markdown="1">
```

Comment:
Can you remove this entire lesson-note? This is a planned removal, as it's kinda redundant at this point, and `e` is explained in the paragraph below.

Reply:
What about the mention about the function being a callback and the callback article?

Reply:
On second thought, I think it would be nice to clarify the definition of a callback. I don't think the article needs to be linked at all though. It basically goes through the same as what the lesson just had written, and I really think that diving deeper with more examples than already given actually might make learners more likely to overcomplicate what a callback is.

What do you think about the lesson note not being removed, but simply its contents being replaced with:
```
When we pass in `alertFunction` or `function (e) ...` as an argument to `addEventListener`, we call this a `callback`. A callback is very simply just a function that is passed into another function as an argument.
```

Reply:
Seems fine to me. We can add the callback article in the additional resources.